modules/ossm-routing-bookinfo-example.adoc

```diff
@@ -4,168 +4,14 @@
 // * service_mesh/v2x/ossm-traffic-manage.adoc
 
 [id="ossm-routing-bookinfo_{context}"]
-= Routing example using the bookinfo application
+= Routing tutorial
 
-The {ProductShortName} Bookinfo sample application consists of four separate microservices, each with multiple versions. Three different versions, one of the microservices called `reviews`, have been deployed and are running concurrently.
+The {ProductShortName} Bookinfo sample application consists of four separate microservices, each with multiple versions. After installing the Bookinfo sample application, three different versions have been deployed and are running concurrently. One of the microservices called `reviews`.  
```

Comment:
```
One of the microservices called `reviews`
```
seems to be hanging. We say that there's a microservice called reviews and don't reference it again.